examples/new-contract1/build/child.main.mjs

```diff
@@ -2,7 +2,7 @@
 /* eslint-disable */
```

Comment:
You should run `./scripts/restore-build.sh` when you accidentally change these files in meaningless ways